modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -65,6 +65,13 @@ export default function UpperToolbar({displayNames, isView, languageIds}) {
 		translations,
 		version,
 	} = useContext(DefinitionBuilderContext);
+
+	function _setAlert(alertMessage, alertType, showAlert) {
```

Comment:
@M-IgorFranca Great job :) Could you just remove the `_` in the beginning of the function name? This is a standard followed on our back-end files, but not used in our front-end files. Thanks!

Reply:
Of course i can, thanks for the feedback.